dev/com.ibm.ws.jpa.tests.jpa_32_fat/test-applications/jakartadata/src/io/openliberty/jpa/data/tests/models/Prime.java

```diff
@@ -34,6 +34,9 @@ public class Prime {
 
     public ArrayList<String> romanNumeralSymbols;
 
+    //For recreating issue 30501, kindly uncomment the following line.
+    // public String romanNumeralSymbols;
+
```

Comment:
30501 indicates that it uses
`    public ArrayList<String> romanNumeralSymbols;`
Switching to `public String romanNumeralSymbols;` is not consistent with 30501.

Reply:
> @njr-11 Test is passing when I give ArrayList

If that is so, then you should be searching for additional ways in which your test case needs to more closely match 30501 (for example, you neglected to make it into an element-collection as 30501 does) rather than forcing a similar error by doing something invalid that doesn't match 30501.  The latter is counterproductive and will only result in EclipseLink saying there is no bug because the way of producing the error isn't valid usage.